README.md

```diff
@@ -118,6 +118,63 @@ This is ideal when you have your own workflows that you want to develop in addit
 
 For example, maybe you have your own existing CLI tools (e.g. using `terragrunt`). In this case, you may want to start by developing your own CLI.
 
+## Install
+
+### OSX
+
+From Homebrew directly
+
+```
+brew install atmos
+```
+
+### cloudposse/packages
+
+Using container
+
+```
+ARG ATMOS_VERSION
+```
+
+If using debian, then `RUN` this
+
+```
+apt-get install atmos@="$(ATMOS_VERSION}-*"
+```
+
+If using alpine, then `RUN` this
+
+```
+apk add atmos@cloudposse~=$(ATMOS_VERSION}
+```
+
+### Go
+
+Get the latest version
+
+```
+go install github.com/cloudposse/atmos
+```
+
+Get a specific version
+
+NOTE: Since the version is passed in via `-ldflags`, the specific version will return `0.0.1` when running `atmos version`.
```

Comment:
```suggestion
__NOTE:__ Since the version is passed in via `-ldflags` during build, when running `go install` without using `-ldflags`, the CLI will return `0.0.1` when running `atmos version`.
```